jsapp/js/components/formEditors.es6

```diff
@@ -761,6 +1159,11 @@ export class AddToLibrary extends React.Component {
       editorState: 'new',
       backRoute: '/library'
     };
+
+    if (this.props.location.pathname === '/library/new/template') {
```

Comment:
@magicznyleszek can you try using the `isActiveRoute` method from the contextRouter mixin here?

Reply:
I did and it's not possible

Reply:
ok.

Reply:
@magicznyleszek, there are people using KPI with a prefix, e.g. `https://my.server/kpi/library/new/template`; would this work with a setup like that?

Reply:
It should be fine for KPI with prefixes, we've been using the same `this.props.location.pathname` construct in `formSubScreens.es6` for a while. I think this is an internal React location prop, independent of `window`.